src/components/menus/operating-status-menu.js

```diff
@@ -53,7 +53,7 @@ const styles = {
     },
 };
 
-const withBranchMenu =
+const withOperatingStatusMenu =
```

Comment:
This menu add actions for equipments of type branch (lines, 2WT,...) on the BaseEquipmentMenu, then I think it could still be named `withBranchMenu`

Reply:
The purpose of this refactoring is to generalize the code, and this menu will be made generic to support additional types, including the 'branch' type